client/coral-framework/hocs/withMutation.js

```diff
@@ -6,7 +6,25 @@ import flatten from 'lodash/flatten';
 import isEmpty from 'lodash/isEmpty';
 import {getMutationOptions, resolveFragments} from 'coral-framework/services/graphqlRegistry';
 import {store} from 'coral-framework/services/store';
-import {getDefinitionName} from '../utils';
+import {getDefinitionName, getResponseErrors} from '../utils';
+import t from 'coral-framework/services/i18n';
+
+class ResponseErrors extends Error {
+  constructor(errors) {
+    super(`Response Errors ${JSON.stringify(errors)}`);
```

Comment:
These are the errors returned from the graph, so you actually get a nice string out of that.